docs/src/pages/[platform]/connected-components/authenticator/trouble-shooting-alert.native.mdx

```diff
@@ -0,0 +1,6 @@
+import { Alert } from '@aws-amplify/ui-react';
+
+<Alert role="none" variation="warning">
+  Please see [troubleshooting](../getting-started/troubleshooting) if you have
```

Comment:
Is there a troubleshooting section for android?

Reply:
This doesn't display on Android or Flutter because `platforms={['web']}` is set in the `index.page.mdx` file -- can delete if you think it's worth it

Reply:
let's remove it until we need it 👍